### PR TITLE
fix: Make ChainVis into basic views

### PR DESCRIPTION
**Pull request by placer14**
Make existing materialized views for ChainVis project into basic views.

### CODE REVIEW

File: storage/migrations/17_chainvis_views_not_materialized.go [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+		main_block.miner,
+		main_block.height,
+		main_block.parent_weight AS parentweight,
+		main_block.timestamp,
+		main_block.parent_state_root AS parentstateroot,
+		parent_block.timestamp AS parenttimestamp,
+		parent_block.height AS parentheight,
+		mp.raw_bytes_power AS parentpower,
+		synced.synced_at AS syncedtimestamp,
+		(SELECT COUNT(*) FROM block_messages WHERE block_messages.block = main_block.cid) AS messages
```
Comment (willscott) [inline storage/migrations/17_chainvis_views_not_materialized.go:29]: name this column 'message_count`?

File: storage/migrations/17_chainvis_views_not_materialized.go [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+	SELECT
+		main_block.cid AS block,
+		bp.parent AS parent,
+		main_block.miner,
+		main_block.height,
+		main_block.parent_weight AS parentweight,
+		main_block.timestamp,
+		main_block.parent_state_root AS parentstateroot,
+		parent_block.timestamp AS parenttimestamp,
+		parent_block.height AS parentheight,
```
Comment (willscott) [inline storage/migrations/17_chainvis_views_not_materialized.go:26]: name as `parent_height` rather than `parentheight`?

**Review by willscott — APPROVED** [review]

File: storage/migrations/17_chainvis_views_not_materialized.go (hunk 2 above)
Comment (willscott) [reply storage/migrations/17_chainvis_views_not_materialized.go:26]: this is an instance of more general: "prefer '_' or camel case over... just pushing the words together"

File: .circleci/config.yml [hunk 3]
```diff
@@ -1,9 +1,6 @@
 version: 2.1
```
Comment (frrist) [inline .circleci/config.yml:1]: This is unrelated to the ChainVis code, yeah?

Comment (placer14) [reply .circleci/config.yml:1]: Correct, these changes only enable the migration test within Circle CI.

File: storage/migrations/17_chainvis_views_not_materialized.go (hunk 2 above)
Comment (placer14) [reply storage/migrations/17_chainvis_views_not_materialized.go:26]: These views go to an externally supported app. Changing column names will require updates from the devs (something I wanted to avoid at the moment to get this change into production for them quickly).
